src/data_info/services/metadata.clj

```diff
@@ -285,6 +289,42 @@
   [data-id {:keys [user]} {:keys [dest recursive]}]
   (metadata-save user (uuidify data-id) (ft/rm-last-slash dest) (boolean recursive)))
 
+(defn- build-ore-uri
+  "Creates a URI for an ORE aggregation from an iRODS path."
+  [path]
+  (str (curl/url (cfg/commons-base) "browse" (string/replace path #"^/+" ""))))
+
+(defn- build-ore [writer agg-path ore-path files avus]
+  (xml/emit (ore/to-rdf (ore/build-ore
+                         (build-ore-uri agg-path)
+                         (build-ore-uri ore-path)
+                         (mapv build-ore-uri (remove (partial = ore-path) files))
+                         avus))
+            writer))
+
+(defn- ore-save
+  "Allows a data commons administrator to save an OAI-ORE file for a data set."
+  [user data-id]
+  (irods/with-jargon-exceptions :client-user user [cm]
+    (validators/user-exists cm user)
+    (let [{:keys [path] :as dir-stat} (stat/uuid-stat cm user data-id)
+          ore-path                    (ft/path-join path "ore.xml")]
```

Comment:
Should we be checking that this file doesn't already exist? As it is I think it'll just blindly overwrite, but maybe that's no big deal (IIRC this is an administrative thing?)

Reply:
I thought that we might want to update these files periodically if the metadata changes, so I didn't add a check to see if the file was already present.